src/services/import-service.spec.js

```diff
@@ -311,4 +311,100 @@ describe('import-service', () => {
 
   });
 
+  describe('getImportBatches', () => {
+    let result;
+
+    beforeEach(async () => {
+      result = await sut.createImportBatch({ importSource, documentsToImport, user });
+      await taskStore.updateOne({ 'taskParams.key': 'v5NSMktadhzquVUAEdzAKg' }, { $set: { processed: true } });
+    });
+
+    it('should return the batch', async () => {
+      result = await sut.getImportBatches();
+
+      expect(result).toEqual([
+        {
+          _id: expect.stringMatching(/\w+/),
+          createdBy: user._id,
+          createdOn: expect.any(Date),
+          completedOn: null,
+          batchType: BATCH_TYPE.importDocuments,
+          batchParams: {
+            name: 'source-1',
+            hostName: 'source1.com',
+            allowUnsecure: false
+          },
+          errors: [],
+          progress: 0.5
```

Comment:
For this value I suggest to not test it indirectly and do a mock of what `getProgressForBatch` returns and just check that whatever the mock returns is put in here.

I would find super useful to actually have a direct test for `getProgressForBatch` (with mocking what the DB returns, of course).

Reply:
This one actually tests that our aggregation pipeline works correctly. I could make it a separate assertion though, to make that more clear.

Reply:
Or at least that 🙏 Cause actually that's the logic that needs testing, otherwise `getImportBatches` itself doesn't do much

Reply:
I'm not sure what the point of mocking would be here, the heart of the logic is in testing that aggregation pipeline, if we mock that out what is the benefit?

Reply:
Please read the comments in order.
You said "...  I could make it a separate assertion though, to make that more clear."
And I said "Or at least that ..."

Reply:
It's true, DB should not be mocked regardless of testing `getProgressForBatch` directly or indirectly via `getImportBatches`. But since you didn't want to go for testing it directly I had agreed to a clear separate assertion.

Reply:
Then I don't understand your initial comment. That getProgress for batch would actually be with an _ it was meant to be "private" (same as what we have in the document-service) since by itself it's pretty useless. I'll rename because I guess that is why we have this confusion.

Reply:
I don't know what you mean by `meant to be "private"` but not worth dragging this so much anymore, I'm fine if you just have a specific assert for the progression value alone.

Reply:
Added the tests :)